source/client-side-encryption/client-side-encryption.md

```diff
@@ -108,8 +108,9 @@ An optional name to identify a KMS provider. Enables configuring multiple KMS pr
 
 **Customer Master Key (CMK)**
```

Comment:
```suggestion
**Master Key**
```

Suggest removing CMK. CMK was AWS specific.

Quoting https://docs.aws.amazon.com/kms/latest/cryptographic-details/basic-concepts.html:
> AWS KMS is replacing the term customer master key (CMK) with AWS KMS key and KMS key.

Suggest replacing other occurrences of CMK with "KMS key".

Reply:
Done - I added a note about CMK because we still use "CMK" in our public documentation for CSFLE: https://www.mongodb.com/docs/manual/core/queryable-encryption/fundamentals/keys-key-vaults/#encryption-keys-and-key-vaults